src/main/app/src/app/zaken/zaak-documenten/zaak-documenten.component.ts

```diff
@@ -109,7 +119,9 @@ export class ZaakDocumentenComponent implements OnInit, AfterViewInit, OnDestroy
     }
 
     ngOnChanges(changes: SimpleChanges): void {
-        this.loadInformatieObjecten();
+        if (changes.zaak && !changes.zaak.firstChange) {
+            this.init(this.zaak, true);
```

Comment:
Dit is wel een mooie oplossing, de ngOnChanges-directive ken ik klaarblijkelijk nog niet goed genoeg, daar ga ik wat meer over lezen.